core/chains/evm/txmgr/txmgr.go

```diff
@@ -86,7 +64,7 @@ type Txm[ADDR types.Hashable[ADDR], TX_HASH types.Hashable[TX_HASH], BLOCK_HASH
 	db               *sqlx.DB
 	q                pg.Q
 	ethClient        evmclient.Client
-	config           Config
+	config           txmgrtypes.TxmConfig[*assets.Wei]
```

Comment:
Use EvmTxmConfig.
Also do this for other places where we are using txmgrtypes.xxx